submissions/alstep07/html-movie-seat-booking/index.html

```diff
@@ -43,334 +43,296 @@ <h2 class="visually-hidden">cinema hall</h2>
                             <span class="seat__icon"></span>
                         </label>
                         <label class="seat">
-                            <input aria-label="row 8 seat 5" type="checkbox" class="seat__input visually-hidden"
-                                id="59">
+                            <input value="row 8 seat 5" aria-label="row 8 seat 5" type="checkbox" class="seat__input visually-hidden">
                             <span class="seat__icon"></span>
                         </label>
                         <label class="seat">
-                            <input aria-label="row 8 seat 6" type="checkbox" class="seat__input visually-hidden"
-                                id="60">
+                            <input value="row 8 seat 6" aria-label="row 8 seat 6" type="checkbox" class="seat__input visually-hidden">
                             <span class="seat__icon"></span>
                         </label>
                         <label class="seat">
-                            <input aria-label="row 8 seat 7" type="checkbox" class="seat__input visually-hidden"
-                                id="61">
+                            <input value="row 8 seat 7" aria-label="row 8 seat 7" type="checkbox" class="seat__input visually-hidden">
                             <span class="seat__icon"></span>
                         </label>
                         <label class="seat">
-                            <input aria-label="row 8 seat 8" type="checkbox" class="seat__input visually-hidden"
-                                id="62">
+                            <input value="row 8 seat 8" aria-label="row 8 seat 8" type="checkbox" class="seat__input visually-hidden">
                             <span class="seat__icon"></span>
                         </label>
                         <label class="seat">
-                            <input aria-label="row 8 seat 9" type="checkbox" class="seat__input visually-hidden"
-                                id="63">
+                            <input value="row 8 seat 9" aria-label="row 8 seat 9" type="checkbox" class="seat__input visually-hidden">
                             <span class="seat__icon"></span>
                         </label>
                         <label class="seat">
-                            <input aria-label="row 8 seat 10" type="checkbox" class="seat__input visually-hidden"
-                                id="64">
+                            <input value="row 8 seat 10" aria-label="row 8 seat 10" type="checkbox" class="seat__input visually-hidden">
                             <span class="seat__icon"></span>
                         </label>
                     </div>
                 </form>
-                <button class="submit-btn" form="tickets" type="submit">Book Now</button>
             </section>
         </main>
+        <aside class="cart">
```

Comment:
Set max height for `.cart` on mobile. Every checked seat move submit button to the bottom 

![image](https://user-images.githubusercontent.com/16196199/109414495-da1b4e80-79bb-11eb-9d6e-2fbe7bd74ce1.png)